scripts/format.js

```diff
@@ -1,15 +1,22 @@
+const path = require('path');
 const chalk = require('chalk');
 
 const prettierWrite = require('../lib/runPrettier').write;
 const prettierConfig = require('../config/prettier/prettierConfig');
 
 const defaultPath = require('../config/prettier/defaultPath');
+const builds = require('../config/builds');
 
 const args = require('../config/args').argv;
 
 console.log(chalk.cyan('Formatting source code with Prettier'));
 
-const filePattern = args.length === 0 ? defaultPath : args;
+const filePattern =
+  args.length === 0
+    ? builds[0].paths.src.map(
+        srcPath => `${path.relative(process.cwd(), srcPath)}/**/*.js`
```

Comment:
Do we want to limit prettier to just JS files?

Reply:
Hmmmm, good point. We weren't limiting this previously. This wasn't a conscious decision.

Reply:
@mbfisher Fixed! 😄